Running-Mastodon/Production-guide.md

```diff
@@ -37,7 +37,9 @@ server {
   server_name example.com;
 
   ssl_protocols TLSv1.2;
-  ssl_ciphers HIGH:!MEDIUM:!LOW:!aNULL:!NULL:!SHA;
+  ssl_ciphers 'ECDHE+CHACHA20:ECDHE+AESGCM'; # or 'ECDHE:!DSS:!PSK:!SRP:!3DES:!RC4:!DES:!IDEA:!RC2:!NULL'
```

Comment:
ECDHE:!… is dangerous. Can reintroduce broken ciphers, because `openssl ciphers` is known for strange behaviour (look for `ECDHE+AES:!RSA` or `EECDH+AESGCM+AES256` for example). Better to use well-known cipher suite with no negation.